server/bin/pbench-index.py

```diff
@@ -114,20 +156,20 @@ def main(options, name):
             " use --config <file> on the command line",
             file=sys.stderr,
         )
-        return 2
+        return error_codes["CFG_ERROR"].value
```

Comment:
Nit: should we change the plural name `error_codes` -> `error_code` as we always refer to one error code everytime

Reply:
sure